utils/utils.go

```diff
@@ -59,11 +59,6 @@ var ErrDetach = errors.New("detached from container")
 
 // CopyDetachable is similar to io.Copy but support a detach key sequence to break out.
 func CopyDetachable(dst io.Writer, src io.Reader, keys []byte) (written int64, err error) {
-	if len(keys) == 0 {
-		// Default keys : ctrl-p,ctrl-q
-		keys = []byte{16, 17}
```

Comment:
I'd prefer it if we kept this around, but set it to the runtime's `DetachKeys` instead of hard-coding C-p C-q. That way, we don't have to unconditionally pass the keys in as part of `StartAndAttach()` - we get the defaults if not overridden.

Reply:
I'm not sure how can I retrieve the runtime DetachKeys here without making an import loop

Reply:
Pass then in as a separate argument, I think (`keys, defaultKeys []byte`)

Reply:
Alternatively, push determining the keys up the stack into `attachContainerSocket()`, which has a runtime available to get the keys

Reply:
Get the default keys, rather

Reply:
I preferred moving the choice in `attachContainerSocket`, so that `term.ToBytes` can be called only once.